catalog/catalog.sh

```diff
@@ -1,50 +1,38 @@
 #!/bin/bash -eu
 
 set +u
-if [ ! -f "$HOME/.catalog.env" ]; then
-	echo "[ERROR] The configuration file ${HOME}/.catalog.env not found"
+
+if [ -z "${CATALOG_URL}" ]; then
+	echo "[ERROR] No value for CATALOG_URL environment variable"
 	echo "Check the Readme.md file for more information"
 	exit 1
 fi
 
-source ~/.catalog.env
-
-if [ -z "${CATALOG_URL}" ]; then
-	echo "[ERROR] No CATALOG_URL property found in the configuration file"
+if [ -z "${CATALOG_PATH}" ]; then
+	echo "[ERROR] No value for CATALOG_PATH environment variable"
 	echo "Check the Readme.md file for more information"
 	exit 1
 fi
-set -u
 
-CUSTOMER=''
-ENVIRONMENT=''
-OPERATION=''
+[ "${ENVIRONMENT}" == "DEFAULT" ] && ENVIRONMENT=""
+[ -z "${CUSTOMER}" ] && CUSTOMER=""
 
-while getopts "c:e:o:" opt; do
-	case $opt in
-	c)
-		CUSTOMER=${OPTARG}
-		echo "Catalog will be generated for customer ${CUSTOMER}"
-		;;
-	e)
-		ENVIRONMENT=${OPTARG}
-		echo "Catalog will be generated for environment ${ENVIRONMENT}"
-		;;
-	o)
-		OPERATION=${OPTARG}
-		;;
-	*)
-		echo "Unsupported option -${opt}"
-		exit 1
-		;;
-	esac
-done
+shopt -s nocasematch
+TASK_TITLE=""
+if [ -z "${TASK_ID}" ] || [[ ! "${TASK_ID}" =~ ^(ta(sk)?-)?[0-9]+$ ]]; then
```

Comment:
Currently, I put just the ID of the task. the regex makes it hard to accept just those starting with ta- or task-

Reply:
These prefixes are optional, ^(ta(sk)?-)**?** Thus, you can use digits directly